src/Utils/Message/ProcessMessage.php

```diff
@@ -131,4 +132,30 @@ public function setImageId($imageId): self
 
         return $this;
     }
+
+    /**
+     * Use search cache.
+     *
+     * @return bool|null
+     *   Defaults to true if not set
+     */
+    public function useSearchCache(): ?bool
+    {
+        return isset($this->useSearchCache) ? $this->useSearchCache : true;
```

Comment:
Default value should not be handled in getter. See above

Reply:
Fixed